assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/resetButtonProps.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useCallback } from 'react';
+import { ButtonProps } from 'antd/lib/button';
+
+type Props = {
+	reset: () => void;
+};
+
+const resetButtonProps = ({ reset }: Props): ButtonProps => ({
+	htmlType: 'reset',
+	onClick: useCallback(
+		(click) => {
+			click.preventDefault();
+			reset();
+		},
+		[reset]
+	),
```

Comment:
No hooks here 😄 , it's better to convert the function to a hook like `useResetButtonProps`, then you can use `useCallback` there.